.github/workflows/node-zxf-deploy-integration.yaml

```
@@ -18,9 +18,18 @@ name: "ZXF: [Node] Deploy Integration Network Release"
 on:
   workflow_dispatch:
     inputs:
-      event:
-        description: JSON representation of the triggering GitHub event
+      ref:
         required: true
+        description: "The github branch or tag that triggered the workflow"
+      author:
+        required: true
+        description: "The author name of the commit"
+      message:
+        required: true
+        description: "The message on the head commit"
+      headSha:
+        required: true
+        description: "The commit ID of the commit that triggered the workflow"
```

I think these should be optional inputs and then we should only trigger the deploy integration job if they're specified.

This way we can do some manual triggers through the workflow dispatch event without calling out to jenkins every time.

Default values: ""

```suggestion
      author:
        required: false
        description: "The author name of the commit"
        default: ""
      message:
        required: false
        description: "The message on the head commit"
        default: ""
      headSha:
        required: false
        description: "The commit ID of the commit that triggered the workflow"
        default: ""

```

---

Also, I think your inputs should be named the same as in the jenkins files... @ElijahLynn @rustyShacklefurd can you please verify that these are the correct inputs:

```
        [defaultValue: '', key: 'ref', regexpFilter: '', value: '$.ref'],
        [defaultValue: '', key: 'author', regexpFilter: '', value: '$.head_commit.author.username'],
        [defaultValue: '', key: 'msg', regexpFilter: '', value: '$.head_commit.message'],
        [defaultValue: '', key: 'sha', regexpFilter: '', value: '$.head_commit.id']
```

I think we want `ref`, `author`, `msg`, and `sha`.

---

Renamed and made defaults empty string.